### PR TITLE
Title changing fixed

### PR DESCRIPTION
**Pull request by Lucitic**
Title changing using page model and migration with seed: be sure to seed the db. 
Uses the Page table to grab the title using section and chapter id.
Pay specific attention to the seed.rb file, application.html.erb, and saasbook_controller.rb.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -208,9 +208,9 @@ GEM
     multi_xml (0.6.0)
     nenv (0.3.0)
     nio4r (2.5.8)
-    nokogiri (1.13.1-x86_64-darwin)
+    nokogiri (1.13.4-x86_64-darwin)
```
Comment (JDDelrosario) [inline Gemfile.lock:211]: Do we still need nokogiri? Or can we remove it?

File: Gemfile.lock [hunk 2]
```diff
@@ -208,9 +208,9 @@ GEM
...
       racc (~> 1.4)
-    nokogiri (1.13.1-x86_64-linux)
+    nokogiri (1.13.4-x86_64-linux)
```
Comment (JDDelrosario) [inline Gemfile.lock:213]: Ditto

File: db/seeds.rb [hunk 3]
```diff
@@ -1,8 +1,162 @@
...
 # frozen_string_literal: true
+
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
 #
 # Examples:
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+Page.new
```
Comment (JDDelrosario) [inline db/seeds.rb:13]: Is this line necessary?

File: Gemfile.lock (hunk 1 above)
Comment (Lucitic) [reply Gemfile.lock:211]: its removed from the gemfile but for some reason when i bundle install it stays in the .lock

File: db/seeds.rb (hunk 3 above)
Comment (Lucitic) [reply db/seeds.rb:13]: genuinely unsure, it was't working without it before but idk if thats still true now

**Review by JDDelrosario — APPROVED** [review]

**Review by ChuanHuoGe — APPROVED** [review]

**Pull request merged**